packages/terra-clinical-site/package.json

```diff
@@ -43,7 +44,9 @@
     "terra-clinical-item-view": "^1.3.0",
     "terra-clinical-label-value-view": "^1.2.0",
     "terra-clinical-no-data-view": "^1.2.0",
+    "terra-clinical-onset-picker": "^0.x",
     "terra-content-container": "^1.0.0",
+    "terra-grid": "^3.0.0",
```

Comment:
Where are we using terra-grid?

Reply:
Not sure how that got added as I'm not using it, removed.